README.md

```diff
@@ -1,3 +1,30 @@
+<<<<<<< HEAD
```

Comment:
Looks like some Merge artifacts have been left over here that probably shouldn't be